core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7453,6 +7453,23 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).withSpark().withLibrary(SqlLibrary.SPARK).ok(expectedSql);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6150">[CALCITE-6150]
```

Comment:
Convert this to `testExtract`, move it to the right place in the file (at the end will collide with other changes), and test at least one other dialect.

Reply:
This is good. Can you break up the strings to make the lines shorter (just one call per line). Put `+` at the start of the line (per our style checker).

Move the method after `testBigQueryDatetimeFormatFunctions`, which is similar. (Anywhere but the end of the file.)

Reply:
> This is good. Can you break up the strings to make the lines shorter (just one call per line). Put `+` at the start of the line (per our style checker).
> 
> Move the method after `testBigQueryDatetimeFormatFunctions`, which is similar. (Anywhere but the end of the file.)

I have fixed the code style and moved `testExtract` after `testBigQuery`. I look forward to your further review. Thank you.